fast_api/routes/organization.py

```diff
@@ -221,8 +220,9 @@ def get_all_organizations(request: Request):
                     "maxRows": org.max_rows,
                     "maxCols": org.max_cols,
                     "maxCharCount": org.max_char_count,
-                    "gdprCompliant": org.gdpr_compliant,
                     "logAdminRequests": org.log_admin_requests,
+                    "conversationLifespanDays": org.conversation_lifespan_days,
```

Comment:
not part of pr but the all-organization request looks quite unoptimized. If it fits we could e.g. change this to a list comprehension & remove the user edges and only return the count of the users (or even count them otherwise since a full get of users only for the count isn't optimal) since only the edges length is used in the frontend